ext/zpages/test/tracez_data_aggregator_benchmark.cc

```diff
@@ -0,0 +1,381 @@
+#include "opentelemetry/ext/zpages/tracez_data_aggregator.h"
+
+#include <benchmark/benchmark.h>
+#include <chrono>
+#include <iostream>
+
+#include "opentelemetry/context/threadlocal_context.h"
+#include "opentelemetry/sdk/trace/tracer.h"
+
+using namespace opentelemetry::sdk::trace;
+using opentelemetry::core::SteadyTimestamp;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext
+{
+namespace zpages
+{
+
+/////////////////////////////// BENCHMARK HELPER FUNCTIONS //////////////////////////////
+
+/*
+ * Helper function that creates and ends num_spans spans instantly. If is_unique is
+ * true, then all spans will have different names.
+ */
+void StartEndSpans(
+    std::shared_ptr<opentelemetry::trace::Tracer> &tracer,
+    int num_spans,
+    bool is_unique = false)
+{
+  opentelemetry::trace::StartSpanOptions start;
+  opentelemetry::trace::EndSpanOptions end;
+  start.start_steady_time = SteadyTimestamp(nanoseconds(0));
+  end.end_steady_time = SteadyTimestamp(nanoseconds(1));
+  for (; num_spans > 0; num_spans--)
```

Comment:
Do you think it's okay given the method arg is pass by value?

Reply:
Yes, because you pass by value the original variable isn't changed. More of a readability thing. In this short function it is easy to follow that num_spans is changed, however if in the future someone adds another loop with num_spans, they can miss that it is changed in the function.

Reply:
okay, sounds good!! i definitely agree using the `int i = 0...` would read better and be good in the case the function changes in the future